{{cookiecutter.project_slug}}/docs/_source/howto.rst

```diff
@@ -0,0 +1,47 @@
+How To - Project Documentation
+======================================================================
+
+Get Started
+----------------------------------------------------------------------
+
+Documentation can be written as rst files in the `{{cookiecutter.project_slug}}/docs/_source`.
+
+{% if cookiecutter.use_docker == 'n' %}
+To build and serve docs, use the command:
+    ::
+    
+        make livehtml 
+    
+from inside the `{{cookiecutter.project_slug}}/docs` directory. 
+{% else %}
+To build and serve docs, use the commands:
+    ::
+    
+        docker-compose -f local.yml build
+        docker run docs
```

Comment:
The [template documentation](https://github.com/hanhanhan/cookiecutter-django/blob/docs/docs/document.rst#L14) has a single command, could we make them consistent with:

```suggestion
        docker-compose -f local.yml up docs
```